README.md

```diff
@@ -370,6 +372,32 @@ The documented advanced deployment scenarios may introduce additional custom env
 | ---------------------------------------------- | ------------------------------------------------------------- |
 | `CORTX_DEPLOY_CUSTOM_BLOCK_STORAGE_CLASS`      | When set to a non-empty string, CORTX will skip the deployment of the `cortx-block-data` Helm Chart, along with the automatic creation of the underlying PersistentVolumes necessary to deploy CORTX. It will instead use the value of this variable as the expected Kubernetes StorageClass and expect all available PersistentVolumes to be created manually by the user. See [Advanced Deployment Scenarios - Using manually-created PersistentVolumes](doc/advanced-deployment-scenarios.md#using-manually-created-persistentvolumes) for use case details.  |
 
+### Deploy with SSL Certificate
+
+By default, CORTX is installed with a default self-signed SSL certificate.  This section describes how to deploy with a specific SSL certificate.
```

Comment:
You don't need 'default' twice so close together here. I would change 'specific' in the second sentence to 'custom'.